tooling/css-variable-docgen-components/src/CharacteristicUsage.tsx

```diff
@@ -32,6 +36,16 @@ export function CharacteristicUsage(
   const characteristicTokenMap =
     getCharacteristics<Record<string, CSSVariable>>(cssVariablesApi);
 
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 2000);
+  }, []);
```

Comment:
unfortunately i think so, I had to add it waiting for the styles to be loaded into the dom otherwise getCharacteristicValue returns null - do you know a better way?